contrib/raftexample/kvstore.go

```diff
@@ -61,7 +67,21 @@ func (s *kvstore) readCommits(commitC <-chan *string, errorC <-chan error) {
 	for data := range commitC {
 		if data == nil {
 			// done replaying log; new data incoming
-			return
+			// OR signaled to load snapshot
+			snapf := filepath.Join(s.snapdir, "db")
+			if fileutil.Exist(snapf) {
```

Comment:
```go
if !fileutil.Exist(snapf) {
    return
}
...
```

always put the short case first. always try to minimize indentation. then we can improve readability. 

thanks!